scripts/release_notes/release_notes.py

```diff
@@ -156,16 +156,18 @@ def extract_notes(commit, seen):
 
     result = {}
 
-    notes = ""
+    notes = message
     if pr is None:
         # Extract PR number from rebase commits if it's not a squashed commit
-        pr, notes = extract_notes_from_rebase_commit(commit)
-    else:
-        # Otherwise, find the release notes section from the squashed commit message
-        match = RE_HEADING.search(message)
-        if not match:
-            return pr, result
-        notes = match.group(1)
+        pr, notes = extract_notes_from_pr(commit)
+        
+    # Find the release notes section from the message
+    match = RE_HEADING.search(notes)
+
+    if not match:
+        return pr, ""
+        
+    notes = match.group(1)
```

Comment:
Why is this not in the `else:` anymore? `notes` from L162 will never be used with this change, no?

Reply:
It's used

Reply:
Ah right, bit confusing how the variable is named and overwritten